google-cloud-datastore/clirr-ignored-differences.xml

```diff
@@ -11,4 +11,9 @@
         <method>com.google.datastore.v1.ReserveIdsResponse reserveIds(com.google.datastore.v1.ReserveIdsRequest)</method>
         <differenceType>7012</differenceType>
     </difference>
+    <difference>
```

Comment:
as we are now on Java 8, please opt for a default method on the interface and remove this exception

Reply:
I am still in favor of adding a default method rather than adding a breaking change here - these other exceptions (I believe) were when we were still on Java 7.

Reply:
Just being the devil's advocate here and contradicting the `clirr` check. How is adding a  new method can be considered a breaking change? We are not breaking any existing expectations of an end user here, Just providing them with more capabilities than the older versions.

Reply:
BTW, i quickly added a dummy default method in the above mentioned interface, and clirr still failed.
<img width="1181" alt="image" src="https://user-images.githubusercontent.com/5915092/191325515-037aa1cc-53a2-4692-9f27-2b0fb7b022a0.png">